### PR TITLE
fix: clovyr fixes

### PR DESCRIPTION
**Pull request by Kodylow**
more fixes from testing today:

- gets rid of console error for null apiUrl
- removes the "all guardians have verified their configs" thing, dprecated behavior by setup leader requirement to be last guy

## Summary by CodeRabbit

- **New Features**
  - Enhanced logic for API announcement matching to ensure both the API URL from the announcement and the provided URL are valid before considering a match.
  - Streamlined user experience in the guardian verification process by replacing the previous message with a spinner that indicates ongoing verification, ensuring clarity and engagement.

### CODE REVIEW

**Comment by Kodylow** [comment]
@tvolk131 or @alexlwn123 can you ack and land this , minor things

File: apps/guardian-ui/src/components/setup/screens/verifyGuardians/VerifyGuardians.tsx [hunk 1]
```diff
@@ -403,9 +403,7 @@ const WaitingForVerification: React.FC<{ verifiedConfigs: boolean }> = ({
 
   return (
     <Flex direction='row' height='100%' my={'auto'} gap={4} align='center'>
-      {verifiedConfigs ? (
-        <Text>{t('verify-guardians.all-guardians-verified')}</Text>
-      ) : (
+      {!verifiedConfigs && (
```
Comment (alexlwn123) [inline apps/guardian-ui/src/components/setup/screens/verifyGuardians/VerifyGuardians.tsx:406]: Nit: This check is redundant, verifiedConfigs is already falsy

**Review by alexlwn123 — APPROVED** [review]
One nit, otherwise lgtm

File: apps/guardian-ui/src/components/setup/screens/verifyGuardians/VerifyGuardians.tsx (hunk 1 above)
Comment (Kodylow) [reply apps/guardian-ui/src/components/setup/screens/verifyGuardians/VerifyGuardians.tsx]: just removed it and put just the spinner up out of that separate component

**Review by elsirion — APPROVED** [review]

**Pull request merged**